source/blog/2014-06-07-jquery-uk-conference-2014.html.markdown

```diff
@@ -40,7 +40,7 @@ After a short break, Veronika went to a workshop about 'automated testing of mob
 
 Back in the main track, [Lea Verou](http://twitter.com/LeaVerou) introduced us to  colourful things. She presented some interesting demos about [The Chroma Zone: Engineering Color on the Web](http://leaverou.github.io/chroma-zone) and showed us how useful CSS colour names can be:
 
-[![Gray, dark gray](https://pbs.twimg.com/media/BnwSZxsIUAACr2-.jpg:large)](https://twitter.com/tomsabin/status/467268696470482944)
```

Comment:
it might be ok to leave this one as is too.  Or maybe there's a better way to embed an image from a tweet these days?